src/Tribe/Views/V2/View.php

```diff
@@ -2562,6 +2576,31 @@ protected function get_repository_args() {
 		return $this->filter_repository_args( $this->setup_repository_args() );
 	}
 
+	/**
+	 * Compiled the global repository args that should be applied to all events queried for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The global filtered repository args.
+	 */
+	protected function get_global_repository_args() {
+		if ( ! is_array( $this->global_repository_args ) ) {
+			/**
+			 * Will filter any repository args to be applied globally on the various repository queries on this view.
+			 *
+			 * @since TBD
+			 *
+			 * @param array<string,mixed> Events Repository args that will be applied globally to all event repository queries.
+			 * @param View $this The View object being rendered.
+			 *
+			 * @return array<string,mixed> The repository args to be applied.
+			 */
+			$this->global_repository_args = apply_filters( 'tec_events_views_v2_view_global_repository_args', [], $this );
```

Comment:
The usual practice, in instances like this one, is to have 2 filters: a generic one (the one already there it's fine) and a specific one for the view slug.
You will see this pattern applied elsewhere in the Views V2 implementation.
I would add a `tec_events_views_v2_{$this->slug}_view_global_repository_args` filter.